### PR TITLE
Make master as default version when package built from master branch

### PR DESCRIPTION
**Pull request by anantpatil**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 else:
     ct = time.gmtime()
     date = "%d%02d%02d%02d%02d" % (ct.tm_year, ct.tm_mon, ct.tm_mday, ct.tm_hour, ct.tm_min)
-    AVI_PIP_VERSION = '17.2b' + date
+    AVI_PIP_VERSION = 'master-' + date
```
Comment (ypraveen) [inline setup.py]: This doesn't conform to python version formatting: https://www.python.org/dev/peps/pep-0440/#version-scheme

Comment (anantpatil) [reply setup.py]: Ok, will fix this.

File: setup.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 else:
     ct = time.gmtime()
     date = "%d%02d%02d%02d%02d" % (ct.tm_year, ct.tm_mon, ct.tm_mday, ct.tm_hour, ct.tm_min)
-    AVI_PIP_VERSION = '17.2b' + date
+    AVI_PIP_VERSION = '18.2.dev-' + date
```
Comment (ypraveen) [inline setup.py]: It can't have any alphabets apart from few designated ones (b for beta, r for release, etc.)

Comment (anantpatil) [reply setup.py]: X.Y.devN is valid right?
https://www.python.org/dev/peps/pep-0440/#developmental-releases

Comment (ypraveen) [reply setup.py]: ah. I missed that. Let's drop the "-" after dev though

**Pull request merged**